### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stephaniewankowicz**
installing qfit on a new computer. Followed the instructions above but got errors with python 3.5 and numpy. I think we should make these dependencies explicit.

### CODE REVIEW

**Comment by blake-riley** [comment]
I haven't tested py 3.5, but it's EOL this year (https://devguide.python.org/#status-of-python-branches) so I agree with putting an exclusion against py 3.5 and below.

Adding numpy to the env early is probably a good idea in keeping problems at bay. 

Did you try with python 3.6? It should work?

**Comment by vdbedem** [comment]
I seem to remember that we avoided openblas-based numpy (default) since that gave inconsistent runtime errors that we were never able to track down. Instead, we relied on mkl. (And its numpy version). This issue may have been resolved since, but just making sure this has been verified to work.

**Comment by blake-riley** [comment]
In each of the CI checks, we do our tests in two separate environments: one mkl (anaconda repos), one openblas (conda-forge repos).

This is not working perfectly, so I cannot answer your question about functionality against openblas.
I have opened #73 to fix down the line.

File: README.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ Once these are installed, you can:
 
 1. Create a new conda env & activate it
    ```bash
-   conda create --name qfit
+   conda create --name qfit --python>=3.7
```
Comment (blake-riley) [inline README.md:27]: Is there a reason that this can't be >= 3.6?

By the way, python 3.8 won't work, because CPLEX doesn't support it yet.

**Pull request merged**